src/plugins/cloudinary/common.js

```diff
@@ -178,17 +178,25 @@ const h264avcToString = (s) => {
   return s;
 };
 
+export const VIDEO_CODEC = {
+  VP9: 'vp9',
+  HEV1: 'hev1',
+  H265: 'h265',
+  H264: 'h264'
+};
+
 const codecToSrcTransformation = (codec) => {
   if (!codec) {
     return {};
   }
+
   switch (codec) {
-    case 'vp9':
-      return { video_codec: 'vp9' };
-    case 'hev1':
-      return { video_codec: 'h265' };
-    case 'h264':
-      return { video_codec: 'h264:baseline:3.0' };
+    case VIDEO_CODEC.VP9:
+      return { video_codec: VIDEO_CODEC.VP9 };
+    case VIDEO_CODEC.HEV1:
+      return { video_codec: VIDEO_CODEC.H265 };
+    case VIDEO_CODEC.H264:
+      return { video_codec: `${VIDEO_CODEC}:baseline:3.0` };
```

Comment:
missing .H264 ? 
if it really is missing, then we're missing a test for this

Reply:
Also, this is very verbose and surly can be refactored to something like:
if (isCodec(...)){
  return {video_codec: codec}
}